### PR TITLE
[OSCD] zer0w

### PR DESCRIPTION
**Pull request by feedb**

### CODE REVIEW

File: rules/windows/process_creation/process_creation_msdeploy.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+title: Msdeploy.exe LOLBIN
```
Comment (w0rk3r) [inline rules/windows/process_creation/process_creation_msdeploy.yml:1]: Consider a more elaborated title, like "Execute Files with Msdeploy.exe"

File: rules/windows/process_creation/process_creation_msdeploy.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+status: experimental
+id: d80d5c81-04ba-45b4-84e4-92eba40e0ad3
+author: Beyu Denis, oscd.community
+date: 2020/10/18
+description: launch binary via msdeploy.exe
```
Comment (w0rk3r) [inline rules/windows/process_creation/process_creation_msdeploy.yml:6]: Consider a more elaborated description, like "Detects file execution using the msdeploy.exe lolbin"

File: rules/windows/process_creation/process_creation_msdeploy.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - https://twitter.com/pabraeken/status/999090532839313408
+tags:
+    - attack.execution
+    - attack.t1218
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ProcessCommandline|contains|all:
```
Comment (w0rk3r) [inline rules/windows/process_creation/process_creation_msdeploy.yml:19]: ProcessCommandline should be replaced with CommandLine to maintain the standard of the process_creation category

File: rules/windows/process_creation/process_creation_msdeploy.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            - '-source:RunCommand'
+            - '-dest:runCommand'
+        Image|endswith:
+            - '\msdeploy.exe'
+    condition: selection
+fields:
+    - ComputerName
+    - User
+    - CommandLine
+    - ParentCommandLine
```
Comment (w0rk3r) [inline rules/windows/process_creation/process_creation_msdeploy.yml:30]: ProcessCommandline should be replaced with CommandLine to maintain the standard of the process_creation category

**Review by w0rk3r — CHANGES_REQUESTED** [review]
Hey @feedb, thanks a lot for the contribution, here are some small changes/small improvements to the rule be ready to merge

**Comment by yugoslavskiy** [comment]
Thank you very much, @feedb! Great work, mate!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**